tests/php72-compatibility.patch

```diff
@@ -530,7 +530,7 @@ diff --git a/tests/input/arrow-functions-format.php b/tests/input/arrow-function
 index d3903ff..8a358e8 100644
 --- a/tests/input/arrow-functions-format.php
```

Comment:
I wonder why there are so many changes 🤔 something is wrong. Should I clean it up?

Reply:
Maybe what I did was wrong? How do you usually proceed? What I did was:
1. edit the patch to remove the hunk about the expected report, so that it applies
2. apply the patch
3. run `vendor/bin/phpcs $(find tests/input/* | sort) --report=summary --report-file=tests/expected_report.txt`

Reply:
@simPod I experienced the same issue, this is why I asked for somebody to finish it. Yes, it seems PHP version running the phpcs is relevant. I guess it also installs different versions of composer dependencies, especially for e.g. 7.2

Reply:
I believe nobody here likes the patches but that's the best we've got. I'll look into it.

Reply:
Yes, there are sniffs that are enabled/disabled by detected PHP version - and it’s intentional here.

Reply:
I forced the php version in step 3 of my process (otherwise the tests would be red)

Reply:
I have opened in as #297 since I cannot push here. We can either merge that (I kept the author) or someone with permissions do some commit shuffling.

Reply:
You mean #298

Reply:
Yes. I have done +1 but somebody opened a PR meanwhile :D